ghcide/session-loader/Development/IDE/Session.hs

```diff
@@ -767,7 +731,7 @@ emptyHscEnv :: NameCache -> FilePath -> IO HscEnv
 emptyHscEnv :: IORef NameCache -> FilePath -> IO HscEnv
 #endif
 emptyHscEnv nc libDir = do
-    env <- runGhc (Just libDir) getSession
+    env <- runGhc (Just libDir) $ getSessionDynFlags >>= setSessionDynFlags >> getSession
```

Comment:
This should be explained a little bit. Doesn't this implicitly call `initUnits` for GHC 9.0 or some other side effects?
What's this supposed to do?